Game/Source/GDKTestGyms/AUptimeGameMode.cpp

```diff
@@ -26,10 +26,18 @@ namespace
 {
 	const FString PlayerDensityWorkerFlag = TEXT("player_density");
 	const float PercentageSpawnpointsOnWorkerBoundary = 0.25f;
```

Comment:
`#include "GDKTestGymsGameInstance.h"`
`#include "Kismet/GameplayStatics.h"`
`#include "NFRConstants.h"`
`#include "Utils/SpatialMetrics.h"`
`const float PercentageSpawnpointsOnWorkerBoundary = 0.25f;`
These include files and variable we doesn't use, can we just remove them?